yarn-project/simulator/src/public/fixtures/opcode_spammer.ts

```diff
@@ -302,19 +299,156 @@ const BYTES_PER_FIELD = Fr.SIZE_IN_BYTES - 1; // 31 bytes of data per field
 const MAX_BYTECODE_BYTES = (MAX_PACKED_PUBLIC_BYTECODE_SIZE_IN_FIELDS - 1) * BYTES_PER_FIELD;
 
 const JUMP_SIZE = encodeToBytecode([new Jump(0)]).length; // JUMP_32
+const INTERNALCALL_SIZE = encodeToBytecode([new InternalCall(0)]).length;
 
 // ============================================================================
 // Type Variant Helpers (for generating multiple configs per opcode)
 // ============================================================================
 
-const ALL_TAGS = Array.from(VALID_TAGS);
-const INT_TAGS = Array.from(INTEGRAL_TAGS);
+//const ALL_TAGS = Array.from(VALID_TAGS);
+//const INT_TAGS = Array.from(INTEGRAL_TAGS);
+// ordered so that limiting #configs per opcode still tests max size (Field)
+const ALL_TAGS = [
+  TypeTag.FIELD,
+  TypeTag.UINT1,
+  TypeTag.UINT8,
+  TypeTag.UINT16,
+  TypeTag.UINT32,
+  TypeTag.UINT64,
+  TypeTag.UINT128,
+];
+
+// ordered so that limiting #configs per opcode still tests max size
+const INT_TAGS = [TypeTag.UINT128, TypeTag.UINT1, TypeTag.UINT8, TypeTag.UINT16, TypeTag.UINT32, TypeTag.UINT64];
 
 /** Build from tag truncating - shorter name */
 function withTag(v: bigint, tag: TypeTag): MemoryValue {
   return TaggedMemory.buildFromTagTruncating(v, tag);
 }
 
+// ============================================================================
+// Random Value Helpers (seeded via SEED env var for reproducibility)
+// ============================================================================
+
+/** Max values for each integer type tag */
```

Comment:
It's not the max value it can take, right? how should we call this? A bit like the difference of supremum and maximum in mathematics